script/dryrun

```diff
@@ -2,13 +2,15 @@
 
 import glob
 import json
+from multiprocessing.pool import ThreadPool
 import os.path
 import sys
 from urllib.request import urlopen, Request
 
-DRY_RUN_URL = 'https://us-central1-moz-fx-data-derived-datasets.cloudfunctions.net/bigquery-etl-dryrun'
-exitcode = 0
-skip={
+
```

Comment:
this looks like an extra newline black would remove

Reply:
I ran black on this. I believe 2 blank lines after imports is standard.

Reply:
apparently not, okay